neo/public/javascripts/map/controllers.js

```diff
@@ -129,7 +151,15 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       $scope.$on("leafletDirectiveMap.dragend", function() {
         if (!$scope.status.init) {
           $scope.bounds = $scope.map.getBounds();
-          var geoData = ($scope.status.logicLevel === 'state') ? $scope.geojsonData.state : $scope.geojsonData.county;
+          var geoData;
+          if($scope.status.logicLevel === 'state'){
+            geoData = $scope.geojsonData.state;
+          } else if($scope.status.logicLevel === 'county'){
+            geoData = $scope.geojsonData.county;
+          }else if($scope.status.logicLevel === 'city'){
+            geoData = $scope.geojsonData.city;
+          }
```

Comment:
we should have an `else` branch, log the error `logicLevel` and set to the default one. (probably the `state` one)